### ISSUE TITLE
pip install error - 'ascii' codec can't decode byte

### ISSUE THREAD

**Issue by tamasbalassa** [mpicbg-csbd/stardist#16]
Hi. 
This might sound very petty, but I've been struggling with this now for a while.

**TLDR**: Previously working installation process fails on another machine with the same setup.

After Ive installed stardist to my notebook without any issues just to try it out for some sample data. I liked it quite much so I've decided to give it a bigger shot and run it on a (local) GPU server.
In both cases I've set up a **python3 virtual environment, simply installed tensorflow-gpu, then stardist, but in the second case it has failed with an error**:

> pip3 install stardist
 Obtaining file:///storage01/balassa/Stardist/stardist
    ERROR: Command errored out with exit status 1:
     command: /storage01/balassa/Stardist/Sd3/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/storage01/balassa/Stardist/stardist/setup.py'"'"'; __file__='"'"'/storage01/balassa/Stardist/stardist/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info
         cwd: /storage01/balassa/Stardist/stardist/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/storage01/balassa/Stardist/stardist/setup.py", line 46, in <module>
        long_description = f.read()
      File "/storage01/balassa/Stardist/Sd3/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 769: ordinal not in range(128)
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

Could anyone help me please what might be the issue here?
In both cases I've tried the process under linux, with python3 and CUDA 9.0. And I have tried to set the env. vars to UTF-8.

**Comment by uschmidt83** [comment]
It might really be a locale issue. Can you [try this](https://stackoverflow.com/questions/51026315/how-to-solve-unicodedecodeerror-in-python-3-6/51027262#51027262)?

**Comment by tamasbalassa** [comment]
Thank You for the answer @uschmidt83!

I did try to set the envs before too, but it was not working for some reason. Then I had to play around abit more, and meanwhile also realized that I was unable to install stardist with gcc-8, but only after switching to gcc-7. Just wanted to note this too.

**Comment by maweigert** [comment]
The latest version (stardist 0.3.4) should have fixed that issue. Installation should now work even if locale is set to non utf-8 encodings.
